eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -23,7 +23,9 @@ steps:
         succeeded(),
         not(eq(variables['${{ parameters.DependenciesVariableName }}'], ''))
       )
-
+  
+  # $(VCPKG_DEFAULT_TRIPLET) added to the restore key of the cache to make it more unique and force a cache miss
```

Comment:
This is true. However, a significant example case we're avoiding is using an [x86 build of curl](https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L95) on an [x64 build ](https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L104) of the SDK libraries. The triplet here lets us capture the target platform architecture, OS (though this is better captured in `$(Agent.Os)`), and linking configuration (static/dynamic) in one variable.  

Please update the comment with the example. This will make the value of `VCPKG_DEFAULT_TRIPLET` immediately more clear to developers who end up here later.

Reply:
> Please update the comment with the example.

@danieljurek can you please give a concrete GitHub suggestion with the example you prefer to be added here.